recipes/mathter/all/conanfile.py

```diff
@@ -0,0 +1,25 @@
+from conans import ConanFile, tools
+import os
+
+class MathterConan(ConanFile):
+    name = "mathter"
+    license = "MIT"
+    homepage = "https://github.com/petiaccja/Mathter"
+    url = "https://github.com/conan-io/conan-center-index/"
+    description = "Powerful 3D math and small-matrix linear algebra library for games and science."
+    topics = ("game-dev", "linear-algebra", "vector-math", "matrix-library")
+    exports_sources = "Mathter/*"
+    no_copy_source = True
+    settings = { "cppstd": ["17", "20"] }
```

Comment:
```suggestion
```

Take a look at `cppcommon` recipe for example to see how to handle a minimum cppstd.

Reply:
Cool, thanks!